src/components/SignupForm/SignupForm.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, fireEvent, cleanup } from '@testing-library/react';
+import { SignupForm } from './SignupForm';
+
+afterEach(cleanup);
+
+it('Inputing text should update email', () => {
+  const utils = render(<SignupForm />);
+  const emailInput = utils.getByTestId('email') as HTMLInputElement;
+
+  fireEvent.change(emailInput, { target: { value: 'example@mail.com' } });
+
+  expect(emailInput.value).toBe('example@mail.com');
+});
+
+it('Inputing text should update passsword', () => {
+  const utils = render(<SignupForm />);
+  const passwordInput = utils.getByTestId('password') as HTMLInputElement;
+
+  fireEvent.change(passwordInput, { target: { value: 'password1234' } });
+
+  expect(passwordInput.value).toBe('password1234');
+});
+
+it('Inputing text should update repeat password', () => {
+  const utils = render(<SignupForm />);
+  const repeatPasswordInput = utils.getByTestId('repeatPassword') as HTMLInputElement;
+
+  fireEvent.change(repeatPasswordInput, { target: { value: 'password1234' } });
+
+  expect(repeatPasswordInput.value).toBe('password1234');
+});
+
+it('Inputing text should update nickname', () => {
+  const utils = render(<SignupForm />);
+  const nicknameInput = utils.getByTestId('name') as HTMLInputElement;
+
+  fireEvent.change(nicknameInput, { target: { value: 'example nickname' } });
+
+  expect(nicknameInput.value).toBe('example nickname');
+});
```

Comment:
It could be nice if you would test other scenarios like:
- typing the same text to password and repeat password then submit
- typing different text to password and repeat password then submit
- You could test validationForm function